src/autogluon_dashboard/plotting/framework_boxplot.py

```diff
@@ -33,8 +33,8 @@ def __init__(
         )
         self.plot = self._box_plot
 
-    def _preprocess(*args) -> None:
-        return Plot._preprocess(*args)
+    def _preprocess(self, **kwargs) -> None:
```

Comment:
this function is not needed as it will be inherited from Plot directly

Reply:
Right